benchmarks/popl18/ple/pos/BasicLambdas.hs

```diff
@@ -43,14 +43,6 @@ id :: a -> a
 id x = x
 
 
-{-@ fmap_id' 
-  :: x:(r -> a)
-  -> {v:Proof | (\r:r -> id (x r)) ==  (\r:r -> (x r) ) } @-}
-fmap_id' :: (r -> a) ->  Proof
-fmap_id' x
-   =   fun_eq (\rrr1 -> x rrr1) (\rrr2 -> id (x rrr2)) (\r -> x  r === id (x r) *** QED)
```

Comment:
I have removed support for alpha equivalence, there is not reason why it passed. 
Before it was ansoundly passing because extensionality was not checking its proof argument at all.